### PR TITLE
Fixed typos, added more questions for some games

### PR DESCRIPTION
**Pull request by asgharkazi**

### CODE REVIEW

File: openfisca_nsw_community_gaming/variables/proceeds_prizes_and_fees.py [hunk 1]
```diff
@@ -95,9 +95,9 @@ class value_of_individual_prize(Variable):
 class no_fees_charged_for_conducting_game(Variable):
     value_type = bool
     entity = Organisation
-    label = """Other than a prize, is there any salary, wage, fee, commission,
-        percentage or other benefit given, paid to or taken by a person in
-        connection with the conduct of the gaming activity?"""
+    label = """Will any entry or other fee be charged to participate
+    in the gaming activity? (other than purchasing goods or services
+    at a normal retail price?)"""
```
Comment (sarafalamaki) [inline openfisca_nsw_community_gaming/variables/proceeds_prizes_and_fees.py]: There shouldn't really be a question mark after price. Perhaps the parenthesis should go before the question mark after activity? It's kind of a funny fragment.

File: openfisca_nsw_community_gaming/variables/proceeds_prizes_and_fees.py [hunk 2]
```diff
@@ -120,7 +120,7 @@ class total_prize_value_from_single_gaming_session(Variable):
 class no_prize_consists_of_money(Variable):
     value_type = bool
     entity = Organisation
-    label = u"None of the prizes consist of or include money"
+    label = u"Does any of the prizes consist of money?"
```
Comment (sarafalamaki) [inline openfisca_nsw_community_gaming/variables/proceeds_prizes_and_fees.py]: Do any of the prizes consist of money? (not Does)

**Review by sarafalamaki — CHANGES_REQUESTED** [review]
Just grammar fixes.

**Review by tjharrop — APPROVED** [review]
Content changes made

**Pull request merged**